Library/Homebrew/formula.rb

```diff
@@ -547,7 +547,7 @@ def head_version_outdated?(version, fetch_head: false)
   # The latest prefix for this formula. Checks for {#head}, then {#devel}
   # and then {#stable}'s {#prefix}
   # @private
-  def installed_prefix
+  def latest_install_prefix
```

Comment:
```suggestion
  def latest_installed_prefix
```

sounds a bit nicer to me

Reply:
I feel like `installed` implies that it returns the currently installed version, but this is not always the case